### DISCUSSION TITLE
GUI Graphical interface suitable for 3.5-inch screens (Interface graphique adapté aux écrans 3.5 pouces.)

### DISCUSSION THREAD

**Discussion by Musclorman** [godotengine/godot-proposals#10708] [Editor]
Hello.

![image](https://github.com/user-attachments/assets/ad8ecce6-c09f-4bcb-9fa7-fbbe21f066ea)

Google Translate - Texte français en bas.
To simplify the use of the editor on devices with small screens or with a low-end touch screen or simply to have large buttons and large texts on normal screens, it would be good to find a way not to have the interface cut off if interface/editor/display_scale is set to 200% or if the device has a very small screen. In the "editor layout" menu, offer other modes than "default". For example "Mobile" where all docks would be hidden and replaced by buttons to display them instead of the 2d/3d/script view on small screens.

Original text - Texte en Français
Pour simplifier l'utilisation de l'éditeur sur des appareils a petits écrans ou avec un tactile bas de gamme ou simplement avoir de gros boutons et de gros textes sur des écrans normaux il serait bon de trouver un moyen pour ne pas avoir l'interface coupé si interface/editor/display_scale est reglé a 200% ou si l'appareil a un ecran tout petit. Dans le menu "disposition de l'éditeur" proposez d'autres modes que "default". Par exemple "Mobile" ou tout les docks seraient masqués et remplacés par des boutons pohr les afficher a la place de la vu 2d/3d/script sur de petits écrans.

Bye.

**Comment by tetrapod00** [comment]
Related: https://github.com/godotengine/godot/issues/95681, https://github.com/godotengine/godot/pull/84114, https://github.com/godotengine/godot-proposals/issues/7156
